README.md

```diff
@@ -12,8 +12,8 @@ The script is realtively simple < 106 lines of code. You can modify it according
 Warning: The author doesn't guarantee anything including the outcome of the running the script. use at your own risk!.
 
 ## Known issue
-* I Never check/tested ratings for individual episode of TV Shows.
-* If you rate a TV show on IMDB. it will mark all season as watched.
+* ~~I Never check/tested ratings for individual episode of TV Shows.~~ This was tested by [@awendland](https://github.com/awendland/trakt-imdb-importer) after commit *455e5cc2*.
```

Comment:
Seems like you solve the issues. This whole section can be removed.

Reply:
I've removed these entries.